### DISCUSSION TITLE
Download filters

### DISCUSSION THREAD

**Discussion by stickycode** [Dynatrace/dynatrace-configuration-as-code#324] [Archive] [answered]
We have two tenancies with many teams supporting their own resources. It would be nice to be able to tag resources and only download resources relevant to my team.

**Comment by tatomir146** [comment, marked as answer]
Hi @stickycode, there is currently some work ongoing and download functionality will be refactored. We still are not planing to support download configs containing only specific tags. But, the product will with settings 2.0 get more fine granular permission management, meaning, you will be able to do this by using different team-specific API keys.

**Comment by stickycode** [comment]
Ok thanks
